shared/src/types/campaign.ts

```diff
@@ -0,0 +1,21 @@
+import { Currency } from './currency';
+import { Timestamp } from './timestamp';
+
+export const CAMPAIGN_FIRESTORE_PATH = 'campaigns';
+
+export type Campaign = {
+	email: string;
+	title: string;
+	description: string;
+	amount_collected_chf: number; // automatically updated by incoming payments.
+	goal?: number;
```

Comment:
This would be the amount in CHF, right? For clarity, I would name it `goal_chf` then.

Reply:
Hmm, I would have thought in the currency of the currency referenced below. 
So that we can display round numbers on the website. The detail page of a campaign would be a fixed currency. Wdyt?